### DISCUSSION TITLE
咋没作用啊，我哪没搞对，大大可以指点一下吗

### DISCUSSION THREAD

**Discussion by XiuiManeia** [jeessy2/ddns-go#803] [Q&A] [answered]
我刷新了很多次ipv6的地址，等了一晚上阿里云上没更新过地址呢
日志里只有
2023/08/02 23:39:29 监听 :9876 ...
2023/08/02 23:40:12 配置文件已保存在: C:\Users\XiuiMane\.ddns_go_config.yaml

**Comment by WaterLemons2k** [comment, marked as answer]
`IPv6` 里面没有填 `Domains`。

**Reply by XiuiManeia** [reply]
哈哈原来是这样，是我没搞懂，谢谢大佬
